tools/content-validation/ci.yml

```diff
@@ -0,0 +1,42 @@
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  DisableComponentDetection: true
```

Comment:
Component detection scanning should probably be turned on here unless we are already scanning the whole repo elsewhere (@weshaggard)

Reply:
Closing this. I see the `archetype-sdk-tool-dotnet.yml` file which probably has all the right configurations in place.